sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -461,6 +463,8 @@ export class MessageSender extends LinkEntity {
           err
         );
         throw err;
+      } finally {
+        this.isConnecting = false;
```

Comment:
Nit: Since we are setting this.isConnecting to false in `finally`, we can remove it from the try block

Reply:
Agreed, this is part of the change I meant when suggesting that `this.isConnecting` be moved into `finally`.

Reply:
Done.